.github/workflows/ci.yml

```diff
@@ -7,29 +7,22 @@ on:
 
 jobs:
   test:
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-24.04
 
     env:
       MIX_ENV: test
 
     strategy:
       # https://hexdocs.pm/elixir/compatibility-and-deprecations.html#between-elixir-and-erlang-otp
       matrix:
-        os: [ubuntu-22.04, ubuntu-24.04]
-        elixir: [1.17, 1.16, 1.15, 1.14]
-        otp: [27, 26, 25]
+        elixir: [1.16, 1.15, 1.14]
+        otp: [26, 25]
         include:
           - elixir: 1.17
             otp: 27
         exclude:
           - elixir: 1.14
             otp: 26
-          - elixir: 1.14
-            otp: 27
-          - elixir: 1.15
-            otp: 27
-          - elixir: 1.16
-            otp: 27
```

Comment:
Elixir 1.17 only works with OTP 27. It's easier to include that pair once rather than exclude all the others.

Reply:
yup, hence why I had the matrix I had. I am not sure I like the include vs exclude in term of readability but I ll go with it for now